### PR TITLE
add @vikkastiwari

### LINKED ISSUE #417: Vikas Tiwari
Capture

### PR DESCRIPTION
**Pull request by vikkastiwari**
resolve #417

### CODE REVIEW

**Review by vikasganiga05 — COMMENTED** [review]
LGTM

File: data.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+  
+  {
```
Comment (AllanRegush) [inline data.json:3]: Please add yourself to the bottom :smile:

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1: Just a minor adjustment :nerd_face:

File: data.json (hunk 1 above)
Comment (vikkastiwari) [reply data.json]: Done!
Updated the required setup.

File: data.json [hunk 2]
```diff
@@ -702,6 +704,7 @@
     "issueNumber": 403
   },
   {
+
```
Comment (schmelto) [inline data.json:707]: remove also this line

File: data.json [hunk 3]
```diff
@@ -1,4 +1,6 @@
 [
+
```
Comment (schmelto) [inline data.json:2]: think you should remove those two lines

**Review by AllanRegush — CHANGES_REQUESTED** [review]
Looking Good :+1: Could you adjust your PR for @schmelto's suggestions?

**Comment by schmelto** [comment]
@vikkastiwari just checking in do you need any help

**Pull request closed without merging**